control/setup_control.go

```diff
@@ -411,3 +416,30 @@ func (p *PIDLoop) StartControlLoop() error {
 
 	return nil
 }
+
+// CreateConstantBlock returns a new constant block based on the parameters
+func CreateConstantBlock(ctx context.Context, name string, constVal float64) BlockConfig {
+	return BlockConfig{
+		Name: name,
+		Type: blockConstant,
+		Attribute: rdkutils.AttributeMap{
+			"constant_val": constVal,
+		},
+		DependsOn: []string{},
+	}
+}
+
+// CreateTrapzBlock returns a new trapezoidalVelocityProfile block based on the parameters
+func CreateTrapzBlock(ctx context.Context, name string, maxVel float64, dependsOn []string) BlockConfig {
+	return BlockConfig{
+		Name: name,
+		Type: blockTrapezoidalVelocityProfile,
+		Attribute: rdkutils.AttributeMap{
+			"max_vel":    maxVel,
+			"max_acc":    30000.0,
+			"pos_window": 0.0,
+			"kpp_gain":   0.45,
+		},
+		DependsOn: dependsOn,
+	}
+}
```

Comment:
how do these look @JohnN193

Reply:
these look good, not sure what the other fields in the trapezoid block mean but if a user really cares about them they can create their own block. Thanks for doing these!

Reply:
yeah I was thinking the same thing